client/app/billing/autoRenew/autorenew.html

```diff
@@ -106,7 +110,7 @@
                             data-on-click="$ctrl.enableBulkAutorenew($ctrl.selectedServices)">
                             <span data-translate="billing_autorenew_service_enable_autorenew"></span>
                         </oui-action-menu-item>
-                        <oui-action-menu-divider ></oui-action-menu-divider>
+                        <oui-action-menu-divider></oui-action-menu-divider>
```

Comment:
Is it really useful though ? There is no menu item below

Reply:
There is more action below ;)
Check https://github.com/ovh-ux/ovh-manager-dedicated/pull/1379/files#diff-87d01eec64e87be9febc1c853f6b41a3R115